lib/email/receiver.rb

```diff
@@ -370,6 +370,7 @@ def process_destination(destination, user, body, elided)
 
         create_topic(user: user,
                      raw: body,
+                     elided: elided,
```

Comment:
one thing that is unclear from the tests here is how this does not add elided unconditionally. 
Can you repeat the first test with SiteSetting.always_show_trimmed_content = false ensuring elided content is not there?